src/css/base.scss

```diff
@@ -16,7 +16,35 @@ body {
   // background-repeat: no-repeat; 
   // background-position: center; 
   // background-size: cover;
-  font-family: 'Concert One', cursive;
+  font-family: 'Shadows Into Light', cursive;
```

Comment:
Looks like we are importing the variables.scss file at the top but its not being used. Can we remove it if we are not going to be using it?